evap/staff/importers.py

```diff
@@ -326,7 +326,12 @@ def _create_user_data_mismatch_warning(user, user_data, test_run):
             msg = format_html(_("The existing user was overwritten with the following data:"))
         return (
             msg
-            + format_html("<br /> - {} ({})", ExcelImporter._create_user_string(user), _("existing"))
+            + format_html(
+                "<br /> - {} ({}) [{}]",
+                ExcelImporter._create_user_string(user),
+                _("existing"),
+                user_edit_link(user.pk),
+            )
             + format_html("<br /> - {} ({})", ExcelImporter._create_user_string(user_data), _("new"))
         )
```

Comment:
I think this is equivalent to
```python
return format_html(
    "{}<br /> - {} ({}) [{}]<br /> - {} ({})",
    msg,
    ExcelImporter._create_user_string(user),
    _("existing"),
    user_edit_link(user.pk),
    ExcelImporter._create_user_string(user_data),
    _("new"),
)
```
Furthermore, you could remove the single argument `format_html` with `msg` above

Reply:
Ahh yes good old `{}<br /> - {} ({}) [{}]<br /> - {} ({})`

Reply:
Actually true ^^